tests/test_extra.lua

```diff
@@ -905,6 +905,111 @@ T['pickers']['buf_lines()']['validates arguments'] = function()
   validate({ scope = '1' }, '`pickers%.buf_lines`.*"scope".*"1".*one of')
 end
 
+T['pickers']['colorschemes()'] = new_set()
+
+local pick_colorschemes = forward_lua_notify('MiniExtra.pickers.colorschemes')
+
+T['pickers']['colorschemes()']['works'] = function()
+  child.set_size(10, 80)
+  child.cmd('colorscheme default')
```

Comment:
All tests already have a fixed color scheme. This makes testing on several Neovim versions possible (as Neovim<0.10 and Neovim>=0.10 have different default color schemes). Is there a particular reason to reset to default here?

Reply:
Yes, the default mini.hues scheme you set in the minimal script does not set `vim.g.colors_name`, so I just chose `default` so it was set correctly.

Reply:
> Yes, the default mini.hues scheme you set in the minimal script does not set `vim.g.colors_name`, so I just chose `default` so it was set correctly.

Then either setting some custom `vim.g.colors_name` in the `pre_case` hook for `T['pickers']['colorschemes()']` set or use `minischeme` should do the trick.

Reply:
Will fix tonight.